### PR TITLE
Theta* algo now able to be chosen for pathfinding

### PR DESCRIPTION
**Pull request by Zach10a**
Implemented the Theta Star algorithm into the menu of the pathfinding layer. In the picture below the Theta Star is in green and the A* is in megenta. This may require a slight tweak as all other factors were the same.

image

### CODE REVIEW

**Comment by aliaksei135** [comment]
Theta* doesn't seem to like non-smooth costmaps. I would stick to using it only on risk maps (see #39 ). 

Please refactor the changed algo name (`RiskThetaStar` -> `GridThetaStar`) properly. I haven't checked all the references to it, but I can see the corresponding tests haven't been refactored!

**Comment by Zach10a** [comment]
That should now work and everything has been refactored

**Comment by Zach10a** [comment]
Bump

File: seedpod_ground_risk/pathfinding/theta_star.py [hunk 1]
```diff
@@ -7,11 +7,21 @@
 from seedpod_ground_risk.pathfinding.a_star import _reconstruct_path
 from seedpod_ground_risk.pathfinding.algorithm import Algorithm
 from seedpod_ground_risk.pathfinding.environment import Node, GridEnvironment
+from seedpod_ground_risk.pathfinding.heuristic import Heuristic, ManhattanHeuristic
 
 
-class RiskThetaStar(Algorithm):
+class GridThetaStar(Algorithm):
```
Comment (aliaksei135) [inline seedpod_ground_risk/pathfinding/theta_star.py:13]: This class doesn't get used anywhere and does not abstract any functionality, so is useless

**Review by aliaksei135 — CHANGES_REQUESTED** [review]

**Pull request merged**